lib/response.js

```diff
@@ -64,10 +64,10 @@ class RESPONSE {
   }
 
   // Adds a header field
-  header(key,value) {
+  header(key,...values) {
     let _key = key.toLowerCase() // store as lowercase
-    value = value !== undefined ? value : '' // default value
-    this._headers[_key] = value // set
+    let _value = values === undefined ? '' : values instanceof Array ? values.length > 1 ? values : values[0] === undefined ? '' : values[0] : values
```

Comment:
I'm thinking we want to store headers as arrays internally, and then decide based on the requestor type (APIG, ALB, Edge) which way to return them.

Reply:
storing the headers as object instead of array allows to access them punctually instead searching for them.
Maybe is possible to use something like an adapter to translate the response in the right format for the requester. What do you think about this?

Reply:
The `headers` would be an object with the header name as the key, but I'm thinking that the values are always stored as arrays.

Reply:
Ok, this make perfectly sense.